app/services/importer_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class ImporterService
+  def initialize(data_set_id, fields: {})
```

Comment:
I'd make this a keyword argument if I were you - it's not clear from context what the `data_set_id` is when it's called. 

Also, the name of this service is deceptively generic - it's not just importing anything, it's importing things from a specific place and saving them as a specific object. Maybe `OrganisationImporterService` instead